savedstate/savedstate/build.gradle

```diff
@@ -6,66 +6,81 @@
  * modifying its settings.
  */
 
-import androidx.build.JetbrainsAndroidXPlugin
+import androidx.build.LibraryType
 import androidx.build.PlatformIdentifier
-import androidx.build.Publish
-import org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
 
 plugins {
     id("AndroidXPlugin")
     id("com.android.library")
+    alias(libs.plugins.kotlinSerialization)
     id("JetbrainsAndroidXPlugin")
 }
 
-JetbrainsAndroidXPlugin.applyAndConfigure(project)
```

Comment:
Binary redirection is still required. Please revert applying this

Reply:
We checked it together. This function does nothing :D  It's empty. I'm not sure why it was declared at all (perhaps it was doing something before, although git history shows it as a new function).

It's enough to just apply the plugin.

Reply:
Let's clean this up in the other modules too then